### ISSUE TITLE
Main window handle not accessible

### ISSUE THREAD

**Issue by PhilLovesToCode** [flyingpie/windows-terminal-quake#97]
This problem just started today.  Whenever I try to launch the app, I get the error "Error Starting Windows Terminal Quake: Main window handle not accessible."  Could it be that there was a Windows 11 update that borked this?  I'm running OS build 22000.160.

**Comment by worthwhileindustries** [comment]
@PhilLovesToCode This happens to me on windows 10 after upgrading windows terminal sometimes and then I have to manually go to the quake exe and launch it. Seems to fix it in that scenario anyway.

**Comment by garf** [comment]
For me, this was happening if my WT app was already launched. Killing the app and then launching WTQ worked.

**Comment by Arakhor** [comment]
Happens to me if I try to add windows-terminal-quake as a startup program. I tried "Run" registry, startup shell folder, task scheduler and task scheduler with 10 sec delay, for some reason the app just refuses to be launched on startup.
I also tried setting launch executable in the program's config to "wt -p Powershell -command windows-terminal-quake", but it seems like the program is unable to pass arguments to the executable?
